packages/react-native-reanimated/src/shareables.ts

```diff
@@ -201,6 +201,14 @@ Offending code was: \`${getWorkletCode(value)}\``);
           if (key === '__initData' && toAdapt.__initData !== undefined) {
             continue;
           }
+          if (key === '__reanimated_secretCodeWrapper') {
```

Comment:
unfortunately I couldn't think any ways without exposing this part however the symbol is wrapped inside this object so it won't be obvious for attacker to search for symbols inside anyway